Dockerfile

```diff
@@ -87,7 +87,7 @@ RUN \
 	tzdata && \
  echo "**** generate locale ****" && \
  locale-gen en_US.UTF-8 && \
- echo -e 'LANG="en_US.UTF-8"\nLANGUAGE="en_US:en"\n' > /etc/default/locale && \
```

Comment:
**-e     enable interpretation of backslash escapes**
but test under `Dockerfile` show bad result. 
```
root@69b361d4ff3e:/# cat /etc/default/locale 
-e LANG="en_US.UTF-8"
LANGUAGE="en_US:en"
```

removing parameter `-e` fix it to 
```
LANG="en_US.UTF-8"
LANGUAGE="en_US:en"
```